x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/schema.tsx

```diff
@@ -217,7 +217,7 @@ export const schema: FormSchema<AboutStepRule> = {
     label: i18n.translate(
       'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.fieldMitreThreatLabel',
       {
-        defaultMessage: 'MITRE ATT&CK\\u2122',
+        defaultMessage: 'MITRE ATT&CK\u2122',
```

Comment:
I noticed this was returning with the literal string when I was doing some testing, probably caused by the big `i18n` update that got merged in a couple weeks ago. This fixes that to display the unicode correctly
![Screenshot 2024-06-10 at 11 20 17 AM](https://github.com/elastic/kibana/assets/56367316/c1fe2ebd-2deb-43bb-98fd-eb68ef474c01)